### DISCUSSION TITLE
How to access the MDMS master data folder from backend  passing tenantId in Query Parameter

### DISCUSSION THREAD

**Discussion by Amitsinghongraph** [egovernments/Digit-Core#271] [DIGIT Platform]
How to access the MDMS master data folder from backend  passing tenantId in Query Parameter
so suppose we two folder pb , pg  so I need to access pb folder master data how to config to access the pb folder. so help me community member.

**Comment by pradeepkumarcm-egov** [comment]
The state-level tenant is declared at the DevOps level. All services will refer to this global variable to decide mdms folder name.  Refer: https://github.com/egovernments/DIGIT-DevOps/blob/4e9b1af299118dabf0fb8ccbd4e3d4a779dd4a72/deploy-as-code/helm/environments/dev.yaml#L28  Here pb added as egov-state-level-tenant-id. All services use this parameter and refer pb folder for master data.

**Reply by Amitsinghongraph** [reply]
Now I am Facing issue naming convention variable name issue like TENANT_TENANTS_MP and city name TENANT_TENANTS_GWALIOR  so how can we resolve this issue . please help community team
